src/components/queryEditor.tsx

```diff
@@ -276,7 +277,7 @@ function AssetTab(props: SelectedProps & { datasource: CogniteDatasource }) {
         },
       });
     }
-  }, [current.externalId]);
+  }, [current]);
```

Comment:
Let's revert to what was here before and add relationships logic on top:

```

  useEffect(() => {
    const relationshipsQuery = current.externalId ? {
      ...query.assetQuery.relationshipsQuery,
      sourceExternalIds: [current.externalId],
    } : null;
    onQueryChange({
      assetQuery: {
        ...query.assetQuery,
        relationshipsQuery
        target: current.value,
      },
    });
  }, [current.value, current.externalId]);
```
Otherwise, it's hard to see how it's affecting the current dashboards..